tests/module_api/test_api.py

```diff
@@ -336,7 +336,10 @@ def test_send_local_online_presence_to(self):
         # Test sending local online presence to users from the main process
         _test_sending_local_online_presence_to_local_user(self, test_with_workers=False)
 
-    @override_config({"send_federation": True})
+    # Default test case disables federation sending. Setting
+    # 'federation_sender_instances' to None turns it back on for the main
+    # process
+    @override_config({"federation_sender_instances": None})
```

Comment:
Similar comments appear in a low of places, maybe we can shorten it:

```suggestion
    # Enables federation sending on the main process.
    @override_config({"federation_sender_instances": None})
```

If someone wants to know *why* this works, they can dig a bit more into how `federation_sender_instances` is used IMO.

Reply:
Gasp! Blasphemy! Let's do it!